provider/datagen/src/transform/cldr/units/mod.rs

```diff
@@ -62,7 +169,59 @@ fn test_basic() {
         .unwrap();
 
     let constants = &und.get().to_owned().constants_map;
+    let ft_to_m = constants.get("ft_to_m").unwrap();
+    let expected_ft_to_m =
+        GenericFraction::<BigUint>::new(BigUint::from(3048u32), BigUint::from(10000u32));
+    assert_eq!(
+        ft_to_m,
+        zerovec::ule::encode_varule_to_box(&ConstantValue {
+            numerator: expected_ft_to_m.numer().unwrap().to_le_bytes().into(),
```

Comment:
what do you mean?

Reply:
The conversion from `BigUint` to `ZeroVec` that you do here should be in a single canonical location, because you'll need it in other places than just this test.

Reply:
I do not need it right now, we can just leave it for the test

Reply:
You will need it soon, and it would make this test more readable.

Reply:
Okay, no problem. 

I am trying to convert from `ConstantValueULE` to `ConstantValue` ... could you help me in that?

Reply:
This is the correct way to do it.